src/rpc.rs

```diff
@@ -440,7 +442,7 @@ impl<T: Runtime> Rpc<T> {
     /// Subscribe to substrate System Events
     pub async fn subscribe_events(
         &self,
-    ) -> Result<Subscription<StorageChangeSet<T::Hash>>, Error> {
+    ) -> Result<ImportedStorageSubscription<T>, Error> {
         let mut storage_key = twox_128(b"System").to_vec();
         storage_key.extend(twox_128(b"Events").to_vec());
         log::debug!("Events storage key {:?}", hex::encode(&storage_key));
```

Comment:
Please remove duplicate `"System Events"` storage key construction, perhaps move into `subscription` where the finalized subscription does.